packages/form-field/src/FormFieldInputContainer/FormFieldInputContainer.styles.ts

```diff
@@ -194,83 +162,56 @@ export const inputWrapperSizeStyles: Record<Size, string> = {
   `,
 };
 
-export const inputWrapperStateStyles: Record<
-  FormFieldState,
-  Record<Theme, string>
-> = {
-  [FormFieldState.Error]: {
-    [Theme.Light]: css`
-      border-color: ${color.light.border.error.default};
-
-      &:hover,
-      &:active {
-        &:not(:focus) {
-          box-shadow: ${hoverRing.light.red};
-        }
-      }
-    `,
-    [Theme.Dark]: css`
-      border-color: ${color.dark.border.error.default};
-
-      &:hover,
-      &:active {
-        &:not(:focus) {
-          box-shadow: ${hoverRing.dark.red};
-        }
-      }
-    `,
-  },
-  [FormFieldState.None]: {
-    [Theme.Light]: css`
-      border-color: ${color.light.border.primary.default};
+export const getInputWrapperStateStyles = ({
+  theme,
+  state,
+}: {
+  theme: Theme;
+  state: FormFieldState;
+}) => {
+  const styleMap = {
+    [FormFieldState.Error]: css`
+      border-color: ${color[theme].border.error.default};
 
       &:hover,
       &:active {
         &:not(:focus) {
-          box-shadow: ${hoverRing.light.gray};
+          box-shadow: ${hoverRing[theme].red};
         }
       }
     `,
-    [Theme.Dark]: css`
-      border-color: ${color.dark.border.primary.default};
+    [FormFieldState.None]: css`
+      border-color: ${color[theme].border.primary.default};
 
       &:hover,
       &:active {
         &:not(:focus) {
-          box-shadow: ${hoverRing.dark.gray};
+          box-shadow: ${hoverRing[theme].gray};
         }
       }
     `,
-  },
-  [FormFieldState.Valid]: {
-    [Theme.Light]: css`
-      border-color: ${color.light.border.success.default};
+    [FormFieldState.Valid]: css`
+      border-color: ${color[theme].border.success.default};
 
       &:hover,
       &:active {
         &:not(:focus) {
-          box-shadow: ${hoverRing.light.green};
+          box-shadow: ${hoverRing[theme].green};
         }
       }
     `,
-    [Theme.Dark]: css`
-      border-color: ${color.dark.border.success.default};
+  };
 
-      &:hover,
-      &:active {
-        &:not(:focus) {
-          box-shadow: ${hoverRing.dark.green};
-        }
-      }
-    `,
-  },
+  return styleMap[state];
 };
 
-export const inputWrapperDisabledStyles: Record<Theme, string> = {
-  [Theme.Light]: css`
+export const getInputWrapperDisabledThemeStyles = (theme: Theme) => {
+  return css`
     cursor: not-allowed;
-    background-color: ${color.light.background.disabled.default};
-    border-color: ${color.light.border.disabled.default};
+    color: ${color[theme].text.disabled
+      .default}; // was not defined in light mode previously
```

Comment:
do we need this comment?

Reply:
Just a note to myself until Chromatic passes